Routes/Routes.go

```diff
@@ -75,7 +75,7 @@ func DeleteStudent(w http.ResponseWriter, r *http.Request) {
 func ReadStudents(w http.ResponseWriter, r *http.Request) {
 	db = sqlconnect.GetMySQLDB()
 	defer db.Close()
-	ss := []models.StudentInfo{}
+	studentslice := []models.StudentInfo{}
```

Comment:
Make it **studentInfo**. Always keep in mind in go while declaring any variable/function etc, Second word of first letter should be capital.

For example: Studentslice -> StudentSlice,